src/tests/ftest/nvme/health.py

```diff
@@ -52,52 +52,47 @@ def test_monitor_for_large_pools(self):
 
         # Create the pools
         self.pool = []
-        for _pool in range(actual_num_pools):
-            self.log.info("-- Creating pool number = %s", _pool)
-            self.pool.append(self.get_pool(create=False))
-            self.pool[-1].scm_size.update(scm_per_pool, "scm_size")
-            self.pool[-1].nvme_size.update(nvme_per_pool, "nvme_size")
-            self.pool[-1].create()
+        for pool_num in range(actual_num_pools):
+            self.log.info("-- Creating pool number = %s", pool_num)
+            try:
+                self.pool.append(self.get_pool(scm_size=scm_per_pool, nvme_size=nvme_per_pool))
```

Comment:
Since we're improving this test, can we remove the use of `self.pool` and instead use a local variable, i.e. `pool_list`?  I do not see this test using any `ServerFillUp` method that makes use of `self.pool`, so this should be a safe change.

Reply:
Tests still passed with that change: https://build.hpdd.intel.com/blue/organizations/jenkins/daos-stack%2Fdaos/detail/PR-9966/6/pipeline